lms/templates/dashboard/_dashboard_entitlement_actions.html

```diff
@@ -0,0 +1,49 @@
+<%page args="course_overview, entitlement, dashboard_index" expression_filter="h"/>
+
+<%!
+from django.utils.translation import ugettext as _
+from django.core.urlresolvers import reverse
+%>
+
+<div class="wrapper-entitlement-action-more">
+
+  ## NOTE: In order for the dropdown functionality provided by lms/static/js/dashboard/dropdown.js to work,
+  ##       this button must include 'id', 'data-dropdown-id', 'data-dropdown-button-id',
+  ##       and class='entitlement-action-more'.
+  <button id="entitlement-actions-dropdown-link-${dashboard_index}"
+          class="entitlement-action entitlement-action-more"
+          type="button"
+          aria-haspopup="true"
+          aria-expanded="false"
+          aria-controls="entitlement-actions-dropdown-${dashboard_index}"
+          data-dropdown-id="#entitlement-actions-dropdown-${dashboard_index}"
+          data-dropdown-button-id="#entitlement-actions-dropdown-link-${dashboard_index}">
+    <span class="sr">${_('Course options for {courseName}').format(courseName=course_overview.display_name_with_default)}</span>
+    <span class="fa fa-cog" aria-hidden="true"></span>
+  </button>
+
+  <div id="entitlement-actions-dropdown-${dashboard_index}" class="entitlement-actions-dropdown" tabindex="-1">
+    <ul id="entitlement-actions-dropdown-list-${dashboard_index}" class="entitlement-actions-dropdown-list" aria-label="${_('Available Actions')}" role="menu">
+      <li id="entitlement-actions-item-unenroll-${dashboard_index}" class="entitlement-actions-item" role="menuitem">
```

Comment:
You don't need to do the '${dashboard_index}' for all of these, instead you can just pass down the container selector 'entitlement-actions-dropdown-${dashboard_index}' and use that to scope these objects. Sort of a DRY for DOM manipulation and styling.